src/NServiceBus.Gateway.RavenDB.AcceptanceTests/SetupFixture.cs

```diff
@@ -8,7 +8,7 @@ public class SetupFixture
         [OneTimeSetUp]
         public void Setup()
         {
-#if NET461
+#if NET472
```

Comment:
kinda wondering whether this is still needed. But can also be addressed separately.